crates/server/src/catalog.rs

```diff
@@ -187,6 +187,24 @@ impl GraphCatalog {
             })
             .collect::<Vec<_>>()
     }
+
+    pub fn remove<K: AsRef<str>>(&mut self, graph_name: K) -> Result<GraphInfo, Status> {
+        self.graphs.remove(graph_name.as_ref()).map_or_else(
+            || {
+                Err(Status::not_found(
+                    "Graph with name '{graph_name}' not found",
```

Comment:
Is this missing a format?

```suggestion
                    format!("Graph with name '{graph_name}' not found"),
```

Reply:
It is .. and not just there 🙈